apps/SoapySDRUtil.cpp

```diff
@@ -278,6 +278,15 @@ static int checkDriver(const std::string &driverName)
  **********************************************************************/
 int main(int argc, char *argv[])
 {
+    //unload any loaded modules when main() scope unwinds
+    struct UnloadModulesScopeExit
+    {
+        ~UnloadModulesScopeExit(void)
+        {
+            SoapySDR::unloadModules();
+        }
+    } unloadModulesInstance;
+
```

Comment:
I wonder if this is worth exposing publicly, maybe with the constructor loading modules?

Reply:
yea I was thinking that at the time. Its probably good idea. Although I cant always have the constructor load